packages/extension/src/view/report/app.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+/**
+ * Internal dependencies
+ */
+import './app.css';
+import {
+  CookiesLandingContainer,
+  CookiesMatrix,
+} from '@ps-analysis-tool/design-system';
+
+const App = ({ data }: { data: any }) => {
```

Comment:
Instead of `App`, should the component name be Report?

Reply:
Is it possible to define the data type, instead of using `any`?

Reply:
This is the top level component for the react app which serves the report and not a component. So no, IMO App makes more sense.